src/app/lib/getComponentInfo.js

```diff
@@ -1,38 +1,49 @@
+let allComponents = [];
 module.exports = (deps) => {
-  let allComponents = [];
   return function getComponentInfo(componentsToFind) {
     const {
       '/config/index': getConfig,
+      '/lib/utils/errors': { NotFoundError },
       fs
     } = deps;
     const config = getConfig();
     const { components: componentsConfig } = config;
     const root = componentsConfig.path;
     const base = componentsConfig.base;
     const componentToFindArr = Array.isArray(componentsToFind) ? componentsToFind : [componentsToFind];
+    const replaceCurrentDir = (dir) => dir.replace(/\/.\//g, '/');
 
     const getComponents = (componentNames) => {
-      return allComponents.filter(component => componentNames.indexOf(component.name)>-1);
+      const components = allComponents.filter(component => componentNames.indexOf(component.name)>-1);
+      if (!components.length) {
```

Comment:
This will only throw the error if none of the components are found. Do we not want to throw the error if any of them are missing?